templates/content/node--oe-project--full.html.twig

```diff
@@ -0,0 +1,47 @@
+{#
+/**
+ * @file
+ * Project full display.
+ */
+#}
+{% if not content.oe_subject.isEmpty() %}
+  {% set _badges = [] %}
+  {% for _badge in content.oe_subject|field_value %}
+    {% set _badges = _badges|merge([{
+      label: _badge|raw,
+      rounded_pill: true,
+    }]) %}
+  {% endfor %}
+{% endif %}
+{{ pattern('content_banner', {
+  background: 'gray',
+  title: label,
+  content: content.oe_teaser,
+  image: image,
+  attributes: create_attribute().addClass(['ps-0']),
+  badges: _badges,
+}) }}
+{% macro field_with_header(field) %}
+  {% if field|field_value is not empty %}
+    <h3 class="fw-bold mb-4 pt-3">{{ field|field_label }}</h3>
+    {{ field }}
+  {% endif %}
+{% endmacro %}
+{% set inpage_navigation_fields %}
+  {{ content.group_project_details }}
+  {{ _self.field_with_header(content.oe_summary) }}
+  {{ _self.field_with_header(content.oe_cx_objective) }}
+  {{ _self.field_with_header(content.oe_cx_impacts) }}
+  {{ _self.field_with_header(content.oe_project_participants) }}
+  {{ _self.field_with_header(content.oe_cx_lead_contributors) }}
+  {{ _self.field_with_header(content.oe_cx_achievements_and_milestone) }}
+{% endset %}
+<div class="mt-md-4-75 mt-4">
+  {{ pattern('inpage_navigation', {
+    title: 'Page content'|t,
+    links: inpage_navigation_links,
+    content: inpage_navigation_fields,
+    full_layout: true,
+  }) }}
+</div>
```

Comment:
wrap these lines (from inpage_navigation_fields to the last line in a block inpage_navigation_content